test/Renci.SshNet.Tests/Classes/Security/Cryptography/Ciphers/AesCipherTest.Gen.cs.txt

```diff
@@ -4,20 +4,20 @@
 // expected encrypted values, and also verifies those values against the .NET
 // BCL implementation as an extra validation before generating the tests.
 
-Dictionary<string, (string, CipherMode?)> modes = new()
+Dictionary<string, (string, string, CipherMode?)> modes = new()
 {
-    ["ecb"] = ("iv: null, AesCipherMode.ECB", CipherMode.ECB),
-    ["cbc"] = ("(byte[])iv.Clone(), AesCipherMode.CBC", CipherMode.CBC),
-    ["cfb"] = ("(byte[])iv.Clone(), AesCipherMode.CFB", CipherMode.CFB),
-    ["ctr"] = ("(byte[])iv.Clone(), AesCipherMode.CTR", null),
-    ["ofb"] = ("(byte[])iv.Clone(), AesCipherMode.OFB", CipherMode.OFB),
+    ["ecb"] = ("mode: null", "iv: null, AesCipherMode.ECB", CipherMode.ECB),
+    ["cbc"] = ("new CbcCipherMode((byte[])iv.Clone())", "(byte[])iv.Clone(), AesCipherMode.CBC", CipherMode.CBC),
```

Comment:
Avoid having to clone/pass IV twice

Reply:
It's cloning for each constructor because the current implementation mutates the IV, but it probably makes more sense just to fix it. I will do that.